client/checkout/blocks/payment-method-label.js

```diff
@@ -36,13 +36,15 @@ export default ( {
 		window.wcBlocksCheckoutData?.storeCountry ||
 		'US';
 
+	const isCreditCard = upeName === 'card';
+
 	return (
 		<>
 			<div className="payment-method-label">
 				<span className="payment-method-label__label">
 					{ upeConfig.title }
 				</span>
-				{ isTestMode && (
+				{ isTestMode && isCreditCard && (
```

Comment:
```suggestion
				{ isCreditCard && isTestMode && (
```
Looks good and working as expected for me. One small suggestion I'd make is we could consider swapping the order of the checks here: we don't even need to check `isTestMode` if `isCreditCard` is already `false`. 🤷 (I suppose this argument can also be switched the other way too--we don't need to check `isCreditCard` if `isTestMode` is `false` either--but `isCreditCard` feels like a more primary condition to me. 😁)